src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
```

Comment:
This code styles only itens with 4 star rating. That means when I changed some random card to stars--2 for example, I don't see the correct styling.
You could solve this by creating separate styles for each case from 1-5 stars, or even better, using the iterators provided by SCSS to do it for you (using @ methods)